src/main/java/IP/duke/command/AddCommand.java

```diff
@@ -0,0 +1,61 @@
+package IP.duke.command;
+import IP.duke.main.DukeException;
+import IP.duke.main.Storage;
+import IP.duke.main.TaskList;
+import IP.duke.main.Ui;
+import IP.duke.task.Deadline;
+import IP.duke.task.Event;
+import IP.duke.task.Task;
+import IP.duke.task.Todo;
+
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a command to add a task.
+ * 
+ * @author Gordon Yit
+ * @Version CS2103T, Semester 2
+ */
+public class AddCommand extends Command {
+    private String addCommand;
+    private boolean isExitCommand;
+
+    /**
+     * Class constructor.
+     * 
+     * @param addCommand the user inputed string to add a task.
+     */
+    public AddCommand(String addCommand) {
+        this.addCommand = addCommand;
+        isExitCommand = false;
+    }
+
+    /**
+     * Executes the command to add a task for Duke.Duke to keep track of.
+     * 
+     * @param tasks lists of tasks
+     * @param ui the user interface.
+     * @param storage the storage file.
+     * @throws DukeException
```

Comment:
Will it be better if you specify when will the exception be thrown?
```suggestion
     * @throws DukeException If ...
```